src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -921,14 +931,14 @@ public static function xpathLiteral(string $s): string
      *
      * The XPath expression should already be processed to apply it in the context of each node.
      */
-    private function filterRelativeXPath(string $xpath): static
+    protected function filterRelativeXPath(string $xpath): static
```

Comment:
should be `@internal`.

Or even better, it should be updated directly to work with the DOM extension if possible (it is already partially updated).

Reply:
I'll add internal annotation. The reason is that `DomCrawler` needs to access this method, but no update is required to make it work the DOM extension.